src/Block/Renderer/DocumentRenderer.php

```diff
@@ -21,9 +21,9 @@
 class DocumentRenderer implements BlockRendererInterface
 {
     /**
-     * @param Document $block
+     * @param AbstractBlock|Document $block
+     * @param HtmlRendererInterface $htmlRenderer
      * @param bool $inTightList
-     *
```

Comment:
Thanks for the merge. I can totally relate to your inner OCD in regards to coding styles ;)